shining-pearl-kazza-palkia-glitchless/route.json

```diff
@@ -0,0 +1,12 @@
+{
+    "title": "Kazza Palkia Route", 
+    "author": [ 
+      {
+          "name": "Cyrun2",
+          "link": "https://twitch.tv/cyrun2"
+      }
+    ],
+    "game": "Shining Pearl", 
+    "generation": "8", 
+    "version": "1.1.2"

```

Comment:
This is meant to be the version of the route, not the version of the game (so something like 1.0.0 for an initial release).